README.md

```diff
@@ -1,6 +1 @@
-# Welcome To The Odin Project!
-
-[The Odin Project (TOP)](https://www.theodinproject.com/) is an open-source curriculum for learning full-stack web development. Our mission is to provide a comprehensive curriculum to learn web development for free. We help users learn the skills and build the impressive portfolio of projects they need to get hired as a web developer.
-
-
-![odin landing image](https://www.theodinproject.com/assets/home-isometric-eebc110218cbdb1460dd2dfc662a4e96ae94747667dc81da422804ef55cfd2d8.svg)
+A special repo for hosting community health files and our ogranisation profiles.
```

Comment:
I know this is just copy+pasted, but is this top-level README required, or could we delete it and just have the `profile` directory?

Reply:
Agreed that the original was low value but I've gone the other way and made it more detailed to document usage since it's not immediately obvious